server/deps.ts

```diff
@@ -1,4 +1,4 @@
-export { opine, json } from 'https://deno.land/x/opine@2.2.0/mod.ts';
+export { opine, json, Router, request, response } from 'https://deno.land/x/opine@2.2.0/mod.ts';
```

Comment:
Would it be easier to just export OpineRequest and OpineResponse? That way you don't have to do `typeof` elsewhere.

Reply:
Sure. I didn't because Deno yelled at me when I tried, saying they're `isolatedModules`. But I can definitely update if that's not a problem.